docs/source/user/first-look-at-DP.ipynb

```diff
@@ -0,0 +1,649 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "scales = np.linspace(0.01, 1., num=100)\n",
+    "epsilons = [make_mean_age(s).map(max_contributions) for s in scales]\n",
+    "accuracies = [laplacian_scale_to_accuracy(scale=s, alpha=0.05) for s in scales]\n",
+    "plt.plot(epsilons, accuracies)\n",
+    "plt.title(\"Epsilon vs. Accuracy at 95% confidence\")\n",
+    "plt.xlabel(\"epsilon\")\n",
+    "plt.ylabel(\"accuracy\");\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When the privacy guarantees becomes looser, the confidence intervals for your estimate become tighter.\n",
+    "Your needs will determine the ideal tradeoff between privacy and utility.\n",
+    "The OpenDP Library provides methods that make this tradeoff as tight as possible."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Composition\n",
+    "We now make a second differentially private release on the teacher survey dataset, to estimate the mean."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "37.35885709620267"
+      ]
+     },
+     "execution_count": 28,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "mean_age_meas = make_mean_age(scale=0.05)\n",
+    "mean_age_meas(csv_data)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When you make multiple releases on the same dataset, the epsilons for each release \"add up\".\n",
```

Comment:
The expenditure for `mean_age_meas` is referenced on a comment on Section 5 (Accuracy vs. Utility). 

```python
# what would the epsilon expenditure be, if the scale were 0.05?
```
Probably nitpicking here, but I'm thinking it could be a good idea to also explicitly mention it's value as we did with the previous $\epsilon = 0.5$ so that the final sum ($\epsilon = 0.65$) becomes more apparent to the reader:

> This second release incurs in an $\epsilon = 0.15$ privacy expenditure.
```python
mean_age_meas.map(max_contributions)
>>> 0.14872014872837347
```